cfgov/cfgov/settings/base.py

```diff
@@ -0,0 +1,89 @@
+import os
+from unipath import Path
+
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+CFGOV_REFRESH = Path(__file__).ancestor(5).child('cfgov-refresh')
```

Comment:
That might make sense for deployment scenarios, but I really think the simplest thing here is assuming both repos are sitting next to each other.